src/frame/mod.rs

```diff
@@ -164,6 +169,9 @@ pub enum Request<'a> {
     /// The second parameter is the vector of values to write to the registers.
     WriteMultipleRegisters(Address, Cow<'a, [Word]>),
 
+    /// A request to report server ID.
```

Comment:
This function code is marked as _Serial Line only_ in the spec. Should we add a comment?

Reply:
Indeed, added the note.